src/cpp/session/SessionModuleContext.cpp

```diff
@@ -928,6 +928,11 @@ FilePath oldScopedScratchPath()
    else
       return userScratchPath();
 }
+
+std::string rLibsUser()
+{
+   return system::getenv("R_LIBS_USER");
```

Comment:
We always fully qualify `core::system` because on some platforms the headers we include define a `system` namespace which results in compiler errors (ambiguous namespace reference)